ppr-ui/src/views/mhrSearch/ConfirmMHRSearch.vue

```diff
@@ -195,7 +196,12 @@ export default class ConfirmMHRSearch extends Vue {
 
   private get feeQuantity (): number {
     // Return selected quantity that is not a combination search
-    return this.getSelectedManufacturedHomes.filter(result => result.selected && !result.includeLienInfo).length
+    // return this.getSelectedManufacturedHomes.filter(result => result.selected && !result.includeLienInfo).length
+    const unique = []
+    this.getSelectedManufacturedHomes.forEach(result => {
+      if (!unique.includes(result.mhrNumber)) unique.push(result.mhrNumber)
+    })
+    return unique.length
```

Comment:
Is`includeLienInfo` no longer a consideration with regards to the fee summary/quantity?
I know it counts combos separately from individual selections, perhaps thats done in another way?

Reply:
Same here as above comment

Reply:
Sounds good, but be wary about leaving too much to resolve after the fact. 
Now that search is in use, there is no more possibilities of partial feature releases.

Reply:
No action required here but will leave comment open for visibility.